analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -41,6 +41,7 @@ import org.ossreviewtoolkit.model.ProjectAnalyzerResult
 import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
+import org.ossreviewtoolkit.model.config.Excludes
```

Comment:
Commit message: Either "excludes" or put "Excludes" into back-ticks if you want to refer to the class.